docs/src/advanced.md

```diff
@@ -0,0 +1,101 @@
+# Advanced Usage
```

Comment:
Should decide on whether to use titlecase or not for particular levels of headings. I made some more comments later on, but may have missed other inconsistencies.